tidb-binlog/handle-tidb-binlog-errors.md

```diff
@@ -37,3 +37,12 @@ Solution: Use the [binlogctl tool](/tidb-binlog/binlog-control.md) to check whet
 ## Data loss occurs during the TiDB Binlog replication
 
 You need to confirm that TiDB Binlog is enabled on all TiDB instances and runs normally. If the cluster version is later than v3.0, use the `curl {TiDB_IP}:{STATUS_PORT}/info/all` command to confirm the TiDB Binlog status on all TiDB instances.
+
+## When the upstream transaction is large, Pump reports an error `rpc error: code = ResourceExhausted desc = trying to send message larger than max (2191430008 vs. 2147483647)`
+
+This error occurs because the gRPC message sent by TiDB to Pump exceeds the size limit. You can adjust the maximum size of a gRPC message that Pump allows by specifying `-max-message-size` when starting Pump.
```

Comment:
```suggestion
This error occurs because the gRPC message sent by TiDB to Pump exceeds the size limit. You can adjust the maximum size of a gRPC message that Pump allows by specifying `max-message-size` when starting Pump.
```